cli/examples/package-authoring/mongodb-enterprise-operator/README.md

```diff
@@ -0,0 +1,25 @@
+# mongodb-enterprise-operator
+
+This example demonstrate how to package [`mongodb-enterprise-operator`](https://github.com/mongodb/helm-charts/tree/main/charts/enterprise-operator) helm chart.
```

Comment:
nit: same as above for "demonstrates" 
This could be changed everywhere I guess

Reply:
Adopted